src/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -812,7 +808,6 @@ public abstract partial class XmlReader : System.IDisposable
         public abstract bool MoveToAttribute(string name);
         public abstract bool MoveToAttribute(string name, string ns);
         public virtual System.Xml.XmlNodeType MoveToContent() { throw null; }
-        [System.Diagnostics.DebuggerStepThroughAttribute]
```

Comment:
Excluding these attributes according to https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/eng/DefaultGenApiDocIds.txt#L16